.nuget/packages.config

```diff
@@ -2,4 +2,32 @@
 <packages>
   <package id="xunit.runner.console" version="2.1.0" />
   <package id="MicroBuild.Core" version="0.3.0" />
+  <package id="Castle.Core" version="3.3.3" targetFramework="net46" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net46" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net46" />
+  <package id="Microsoft.AspNet.WebApi.OData" version="5.7.0" targetFramework="net46" />
+  <package id="Microsoft.Data.Edm" version="5.8.4" targetFramework="net46" />
+  <package id="Microsoft.Data.OData" version="5.8.4" targetFramework="net46" />
+  <package id="Microsoft.Web.Xdt" version="2.1.1" targetFramework="net46" />
+  <package id="Moq" version="4.5.21" targetFramework="net46" />
```

Comment:
Why are these here? This packages.config is used to pull down command-line tools or build stuff and should not have project-specific packages.

Reply:
Somehow, project packages.config has trouble in  installing packages. As workaround, I think install packages here might resolve this problem.

Reply:
Are there alternative workaround? I'm not sure if I understand the problem and solution. Maintaining this list along with the packages.config in each project seems like a headache.